docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -59,4 +59,35 @@ For example:
 
 ----
 minishift start --iso-url centos
-----
\ No newline at end of file
+----
+
+[[x509-certificate-is-valid-for-10.0.2.15-127.0.0.1-192.168.99.100-not-192.168.99.101]]
+== x509 certificate is valid for 10.0.2.15, 127.0.0.1, 172.17.0.1, 172.30.0.1, 192.168.99.100, not 192.168.99.101
+
+If you started a stopped {project} VM then due to IP change you might hit following x509 certificate error:
```

Comment:
No need to mention "due to the IP change" here. The reason for the error can be explained in greater detail later.

I'd also replace "hit" with another word (such as "encounter"); "hit" has a more conversational tone. Similarly, I would use "X.509" instead of "x509".

There's also tense confusion between "if you started" and "you might encounter". I'd recommend rewriting this as follows:

"Starting a stopped {project} VM can produce the following X.509 certificate error:"